ansible/roles/backstage/tasks/main.yml

```diff
@@ -4,10 +4,13 @@
     name: backstage
   become: yes
 
+- name: install boto3
+  command: "sudo pip install boto3"
+
 - name: Setup ssh credentials
-  s3:
+  aws_s3:
```

Comment:
this is probably a better way to refer to it, but `s3` is an alias for `aws_s3` so I don't think it would have fixed anything. Was this more of a cleanup thing or were you having problems getting this to work?